frontend/src/components/assessment-creation/CreateAssessmentHeader.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import {
+  Flex,
+  VStack,
+  Text,
+  Button,
+  Box,
+  Spacer,
+  HStack,
+  IconButton,
+} from "@chakra-ui/react";
+import {
+  TextOutlinedIcon,
+  SaveOutlinedIcon,
+  EyeOutlinedIcon,
+  MoreVerticalOutlineIcon,
+  ArrowBackOutlineIcon,
+} from "../../assets/icons";
+
+interface CreateAssessementHeaderProps {
+  assessmentName: string;
+  date: string;
+}
+
+const CreateAssessementHeader = ({
+  assessmentName,
+  date,
+}: CreateAssessementHeaderProps): React.ReactElement => {
+  return (
+    <Box
+      margin="-1.5em -2em 2em -2em"
+      padding="1.5em 2em 1.5em 2em"
+      borderBottom="1px"
+      borderBottomColor="grey.200"
+    >
```

Comment:
once you remove the navbar from this page, you should be able to remove these extra css attributes as well!